docs/cli/discount-code-exporter.md

```diff
@@ -9,11 +9,15 @@ The constructor accepts two arguments:
 * A required object containing the following values:
   * `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
   * `batchSize` (Number): Amount of codes not more than 500 to process concurrently (Optional. Default: 500)
+  * `language` (String): Language used for localised fields if no template is given (Optional. Default: `'en'`)
   * `accessToken` (String): Access token to be used to authenticate requests to API. Requires scope of [`view_orders`]
   * `delimiter` (String): CSV delimiter (Optional. Default: `','`)
   * `multiValueDelimiter` (String): CSV delimiter used in multivalue fields (Optional. Default: `';'`)
   * `exportFormat` (String): Export format ['csv', 'json'] (Optional. Default: 'json')
   * `predicate` (String): Query string specifying (where) predicate. More info on predicates [here](https://docs.commercetools.com/http-api.html#predicates) (Optional)
+  * `headerFields` (Array<String>): An array of column names the exported CSV file should contain. This headerFields array should contain the required columns of the CSV file(Optional. If omitted, a default set of column fields is used. Currently, these fields are: `name`, `description`, `code`, `cartDiscounts`,`cartPredicate`,`groups`,`isActive`,`validFrom`,`validUntil`,`references`,`maxApplications`,`maxApplicationsPerCustomer`.
```

Comment:
If we use the term `headerFields` in other packages all good. I felt `fields` might already be enough.